content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -268,13 +269,14 @@ using an external CRI implementation.
 
 ### Using internal IPs in your cluster
 
-In order to set up a cluster where the master and worker nodes communicate with internal IP addresses (instead of public ones), execute following steps.
+In order to set up a cluster where the control-plane and worker nodes communicate with internal IP addresses
+(instead of public ones), execute following steps.
 
 1. When running init, you must make sure you specify an internal IP for the API server's bind address, like so:
 
-   `kubeadm init --apiserver-advertise-address=<private-master-ip>`
+   `kubeadm init --apiserver-advertise-address=<private-control-plane-node-ip>`
 
-2. When a master or worker node has been provisioned, add a flag to `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` that specifies the private IP of the worker node:
+2. When a control-plane or a worker node has been provisioned, add a flag to `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` that specifies the private IP of the worker node:
```

Comment:
Silly question: is there a particular reason to suggest to change the drop-in file instead of using the node registration object in the kubeadm config file?

Reply:
i'm not the author of this and this seems outdated.
i'm going to remove it. any objections?

Reply:
Not from me, we definitely don't want to recommend users modify a systemd drop-in file.

Reply:
+1 nuke it